database/change-20211017-AB.sql

```diff
@@ -11,10 +11,10 @@ VALUES ('Admin'),
 CREATE TABLE User
 (
     UserID      INT PRIMARY KEY AUTO_INCREMENT,
-    Username    TEXT UNIQUE,
+    Username    VARCHAR(20) UNIQUE,
     Name        TEXT,
     Password    TEXT,
-    Email       TEXT UNIQUE,
+    Email       VARCHAR(50) UNIQUE,
```

Comment:
You didn't rename the file as you should

Reply:
Why should the file be renamed? I only updated the type to a limited string as MySQL requires all `UNIQUE` fields to be limited. The file can be updated now, before it's run on production, afterwards new change files will have to be created to update stuff like this.